pkg/build/api/types.go

```diff
@@ -63,6 +63,31 @@ const (
 	// forces the build to be processed by the build controller queue without waiting
 	// for a resync.
 	BuildAcceptedAnnotation = "build.openshift.io/accepted"
+
+	// BuildCreatedEvent is the reason associated with the event registered when a build is created.
+	BuildCreatedEventReason = "BuildCreated"
```

Comment:
we don't have to display them all in the console by default, but is that a good reason not to create them?

I viewed all 3 as interesting primarily in an unhealthy system.  
Created - a build was kicked off for some reason (manual, image trigger, git push, etc) - i think i want to know this.

Started - The system is attempting to start working on my build.. particularly relevant if you are doing serial builds, there could be a long time between creating a build and starting it.

Running - In an overscheduled system, there could be a long time between starting a build(pod is created) and the pod getting scheduled.  I like to know when my build is actually running, not when it was eligible to run.

I can maybe see the argument for leaving out "Running" because in a healthy system started and running will happen very close together, but i definitely thing created and started are distinct and important due to the serial build policy.